Azure WAF Enhancement (#39344)
demisto/content: Packs/AzureWAF/README.md

### Supported Timestamp Formats:

1. yyyy-MM-ddThh:mm:ssZ 
2. MMM dd yyyy HH:mm:ss

For *msft_azure_waf_raw*, timestamp ingestion is according to the fields below in UTC (00:00) time zone. 

- timeStamp
- timeStamp_t
- time
- TimeGenerated

Examples:

- 2025-03-26T05:39:46Z
- 2024-11-19T10:50:39+05:00
- Nov 19 2024 12:50:39

> [!NOTE] 
> Time offsets from UTC is supported. 
>
> See [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) for more information.

***

## Data Collection

To configure Microsoft Azure WAF to send logs to Cortex XSIAM, follow the below steps.

### Prerequisites

- Create an **Azure event hub**. For more information, refer to Microsoft's official [documentation](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
- Make sure that you have at least a Security Administrator role.
- For Azure WAF Front Door event logs you will need to [Create an Azure Front Door profile](https://learn.microsoft.com/en-us/azure/frontdoor/create-front-door-portal?tabs=quick)

### Stream logs to an event hub

1. Sign in to the **Microsoft Entra admin center**.
2. Navigate to **Identity** &rarr; **Monitoring & health** &rarr; **Diagnostic settings**.

>     - If you wish to stream Front Door logs, select the relevant Front Door profile 
>     - Within the profile, navigate to Monitoring, select Diagnostic Setting.

Reviewer: ```suggestion
    - To stream Front Door logs, select the relevant Front Door profile 
    - Within the profile, navigate to **Monitoring**, and select **Diagnostic Setting**.
```

<~XSIAM>

# Azure Web Application Firewall 

> <~XSOAR> [Azure WAF Integration](https://xsoar.pan.dev/docs/reference/integrations/azure-waf) </~XSOAR>

Reviewer 2: The <~XSOAR> tag cannot be inside the <~XSIAM> tag, lets rearrange it :)

Author: The following pack has the same tags:
1. https://cortex.marketplace.pan.dev/marketplace/details/GoogleCloudSCC/
2. https://fusion-qa-37.xdr-qa2-uat.us.paloaltonetworks.com/market/browse/GoogleCloudSCC

Author: OK, moving the tag below description is the best practice? (xsoar and xsiam will have title and overview)
